import logging
from fastapi import HTTPException

def new_verify_phone_and_email():
    logging.info('fn new verify phone and email: called')
